catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/table/row.view.js

```diff
@@ -63,7 +63,9 @@ module.exports = Marionette.LayoutView.extend({
         this.handleResultThumbnail();
     },
     handleResultThumbnail: function() {
-        if (this.model.get('metacard').get('properties').get('thumbnail')) {
+        var hiddenColumns = user.get('user').get('preferences').get('columnHide');
+        if (this.model.get('metacard').get('properties').get('thumbnail') &&
```

Comment:
This looks like part of a different PR; was this included intentionally?

Reply:
I didn't create a separate branch. For that I assumed my last PR would be merged already.